src/test/java/io/quarkus/bot/release/ReleaseInformationTest.java

```diff
@@ -0,0 +1,34 @@
+package io.quarkus.bot.release;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.aggregator.ArgumentsAccessor;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import io.quarkus.bot.release.util.Branches;
+
+public class ReleaseInformationTest {
+
+    @ParameterizedTest
+    // version,branch,qualifier,firstFinal,expectedResult
```

Comment:
I'm trying to wrap my head around what this `firstFinal` means. IIUC, it means that either the version ends with `.0`, or it ends with `.1` but there is no corresponding `.0` (because the original release failed or something)?

Reply:
I think this would deserve a (JavaDoc) comment  in the `ReleaseInformation` class

Reply:
Yes exactly.

Fair point about needing more Javadoc, I'll do that in another PR.